src/index.jsx

```diff
@@ -108,14 +82,6 @@ function Form() {
 
         <Heading sx={{ fontSize: 2, mb: 1, mt: 3 }}>Open in</Heading>
 
-        {/* <DropdownMenu
-            items={menuItems}
-            onChange={handleMenuChange}
-            placeholder={AVAILABLE_APPS[app].title} // Display the value of 'name' or a placeholder
-            open={isDropdownOpen}
-            onOpenChange={changeOpenState}
-        /> */}
-
         <div class="select-container">
             <select class="custom-select" value={app} onChange={handleSelectChange}>
```

Comment:
Now uses the <select> element instead of Primer's version. Styling is also in a separate file ie no inline styles.